src/components/InstallQryptoPrompt/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { withStyles, Typography, Button, Popover, Grid } from '@material-ui/core';
+import { inject, observer } from 'mobx-react';
+import { FormattedMessage } from 'react-intl';
+import cx from 'classnames';
+
+import styles from './styles';
+
+
```

Comment:
nit: extra space, is it intentional to have 2 lines after the last import?

Reply:
Alex told me before the double line after the imports was standard, but not sure if that was his code style or if its React standard. I prefer 1 line too, will remove all.